### PR TITLE
(fix) case sensitive parameters in policies

### PR DESCRIPTION
**Pull request by steph409**
# Overview/Summary

Replace this with a brief description of what this Pull Request fixes, changes, etc.

## This PR fixes/adds/changes/removes

As discussed in issue #185, the parameter names are case sensitive for terraform, and it would be great if the policies would be consistent in the casing of the parameter names.

### Breaking Changes

No breaking changes.

## As part of this Pull Request I have

- [X] Read the Contribution Guide and ensured this PR is compliant with the guide
- [X] Checked for duplicate [Pull Requests](https://github.com/Azure/azure-monitor-baseline-alerts/pulls)
- [X] Associated it with relevant [GitHub Issues](https://github.com/Azure/azure-monitor-baseline-alerts/issues) or ADO Work Items (Internal Only)
- [X] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/azure-monitor-baseline-alerts/)
- [X] Ensured PR tests are passing
- [X] Updated relevant and associated documentation (e.g. Contribution Guide, Docs etc.)

### CODE REVIEW

**Comment by Brunoga-MS** [comment]
LGTM

**Review by Brunoga-MS — DISMISSED** [review]
LGTM

File: services/Cdn/profiles/Deploy-CDNP-OriginLatency-Alert.json
Comment (Brunoga-MS) [inline services/Cdn/profiles/Deploy-CDNP-OriginLatency-Alert.json:1]: Looks good to me ... I would just update the display name on line 8 and 9 too

Comment (steph409) [reply services/Cdn/profiles/Deploy-CDNP-OriginLatency-Alert.json:1]: hi, just saw your message, just updated it

**Review by arjenhuitema — APPROVED** [review]
LGTM

**Pull request merged**